assets/activation-scripts/activate

```diff
@@ -94,11 +104,72 @@ start_services_blocking () {
   fi
 }
 
+# Parse command-line arguments.
+# shellcheck disable=SC1091
+OPTIONS=c:
+LONGOPTS=turbo,noprofile
```

Comment:
add a `--command` longoption for `-c`?

Reply:
Hmmm .. the idea of `-c` is to be compatible with shells, and none of `bash`, `zsh` or `tcsh` support the `--command` arg - only `fish` supports that. So on that basis it wouldn't hurt, but as it's not a universally-supported shell option it doesn't really fit with our goal for the `activate` script to emulate the function of a shell.

So .. bottom line I wouldn't object, but don't think it's necessary either.

Reply:
I went ahead and added the option as I squashed the last commits.